tests/unit/experiment_api/test_request_exception_wrapper.py

```diff
@@ -49,4 +51,19 @@ def f():
     with Scenario() as s:
         s.config.comet_url() >> COMET_CLOUD_URL
         with pytest.raises(exceptions.CometLLMException):
-            f()
\ No newline at end of file
+            f()
+
+
+def test_wrap__request_exception_non_llm_project_sdk_code__log_specifc_message_in_exception():
```

Comment:
Since handle_response is going to be moved to another module and is no longer the responsibility of the wrapper, its call should be mocked in the unit tests.